Tests/Data/Elliptic/circle_radius_1/circle_1e1_axi.prj

```diff
@@ -51,7 +51,7 @@
         </processes>
         <output>
             <type>VTK</type>
-            <prefix>circle_1e1_axi</prefix>
+            <prefix>circle_1e1_axi_pcs_{:process}_ts_{:timestep}_t_{:time}</prefix>
```

Comment:
Do we have a test with precision for time and formatting (leading zeros) for the timestep?

Reply:
The current tests don't have formatted time other than the default formatting. At the moment the time step is nowhere filled up with leading zeros.

Reply:
Added a unit test for this case.